### ISSUE TITLE
Create a video on distributed leadership

### ISSUE THREAD

**Issue by abbycabs** [mozilla/open-leadership-training-series#12]
Notes for video:

VIDEO: Distributed Leadership
- Story of how a contributor moved through a project to take on a new role
- What this meant for the person, and the project

Maybe @mbonsma might be interested?

Full article: https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/understand-meaningful-participation-and-distributed-leadership/

Edit here: https://github.com/mozilla/open-leadership-training-series/blob/gh-pages/_articles/building-communities-of-contributors/understand-meaningful-participation-and-distributed-leadership.md#distributed-leadership

**Comment by mbonsma** [comment]
Yeah I'm interested! Could this be from the study group perspective?

**Comment by abbycabs** [comment]
Yes, study groups welcome! Any open project 😄

**Comment by abbycabs** [comment]
Tips on video creation: https://docs.google.com/document/d/11nGEQ3vuy7GtQwncb5MbDTAehLipC3kENT5zNF6QufY/edit?usp=sharing

**Comment by mbonsma** [comment]
I won't get to this before Mozfest, but I'll try to finish by mid-November!

**Comment by abbycabs** [comment]
👌
